datadayessentials/config/__init__.py

```diff
@@ -1,12 +1,9 @@
 """
 This module provides classes for managing the core local config file, including for synchronizing the config file with a remote version stored in a machine learning workspace.
 """
-from ._config import LocalConfig, GlobalConfig
-from ._config_setup import ConfigSetup
+from ._config import CloudProviderConfig
```

Comment:
Id rename this to just Config - as 'CloudProviderConfig' relates to the implementation rather than the functionality that the class provides